certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -271,14 +316,20 @@ SETUP
  USEROPTIONSCHR(4)         = "Renew certificate"
  USEROPTIONSCHR(5)         = "Disburse funds by check"
  USEROPTIONSCHR(6)         = "Suspend certificate"
- DEBUGMODE                 = FALSE
  SYMXINSTANCE              = @SYMXINSTANCEID
  SYMXCLIENTNUM             = @CLIENTNUMBER
 
- FOR TMPLOOP=1 TO INDENTMAX
+
+ FOR BNOLOCALLOOP=1 TO BNOINDENTMAX
   DO
-   INDENT(TMPLOOP)=REPEATCHR(" ",TMPLOOP*2)
+   BNOINDENT(BNOLOCALLOOP)=REPEATCHR(" ",BNOLOCALLOOP)
```

Comment:
I think the second BNOLOCALLOOP on line 325 needs to have '*2' after it like your previous code, and other Banno Powerons.
Here is the Previous code:
INDENT(TMPLOOP)=REPEATCHR(" ",TMPLOOP*2)

Reply:
As this serves only for "prettier" JSON output (and a little easier to read) I'm going to leave this as is.  There's a reason why I left it as single spacing for the indents instead of double-spacing... I just can't remember off the cuff and I don't want to have to retest everything.  I'm guessing that there were areas of the debug output where the output was coming too close to the 132 character cutoff and , as you know, any random cut-off of the end of a line of JSON will hose the whole thing.